### ISSUE TITLE
A11y_Azure Tools for VS Code [BM]_Default Screen_Screen Reader: NVDA/Narrator is not reading the status message "Enabled" when we press enter on "Enable" button.

### ISSUE THREAD

**Issue by Accessibility56** [microsoft/vscode-azurefunctions#2675]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
#Azure Tools for VS Code; #A11ySev2; #A11yTCS; #BM-Azure Tools for VS Code-Win-Feb2021; #DesktopApp; #Screen Reader; #WCAG4.1.3; #Status Messages; #NVDA; #Win10; #Narrator;

### OS Details:
Windows 10 Version 20H2 (OS Build 19042.685)

### Environment Details:
Application Name: Azure Tools for VS Code
Version: 1.53.1
Commit: 5d424b828ada08e1eb9f95d6cb41120234ef57c7
Date: 2021-02-08T23:29:42.785Z
Electron: 11.2.1
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Windows_NT x64 10.0.19042
Azure Function Version: 1.2.1

### **Repro Steps:**

1. Open Visual studio code
2. Turn on NVDA/Narrator
3. Tab till extension button and press enter on it.
4. Search for "Azure Functions" and select it.
5. Tab till "Install" button and press enter on it.
6. Tab till "Azure" button from left side pane and press enter on it.
7. Default screen will get open.
8. Tab till "Enable" button and press enter on it.
9. Verify NVDA/Narrator is reading "Enabled" message or not.

### Actual Result:
NVDA/Narrator is not reading the status message "Enabled" when we press enter on "Enable" button.

### Expected Result:
NVDA/Narrator should read the status message "Enabled" when we press enter on "Enable" button.

### User Impact:
Screen Reader users will not get the complete information on screen if screen reader is not reading it.

### Attachment for Reference:

![2675_Narrator issue](https://user-images.githubusercontent.com/38523250/108309916-7a14f300-71d8-11eb-8a3a-73172e5b0087.png)
[2675_Status Message_NVDA.zip](https://github.com/microsoft/vscode-azurefunctions/files/6020012/2675_Status.Message_NVDA.zip)

**Comment by ejizba** [comment]
Our extension does not own or control the marketplace. This would reproduce for any VS Code extension and should be filed directly on VS Code instead: https://github.com/microsoft/vscode/issues
